fhirr5/ballerina/src/main/resources/fhirservice/utils.bal

```diff
@@ -11,6 +11,17 @@
 // specific language governing permissions and limitations
 // under the License.
 import ballerinax/health.fhir.r5;
+import ballerinax/health.fhir.r5.parser;
+import ballerina/http;
+import ballerina/log;
+
+isolated http:Client? conditionalInvokationClient = ();
```

Comment:
Can we keep this conditionalInvokationClient in Listener class and pass to the preprocessor in the init()?